docs/docs/user-guides/reverse-tunnel.md

```diff
@@ -0,0 +1,87 @@
+---
+layout: default
+title: Node to Node Connections (Reverse Tunnel)
+nav_order: 7
+parent: User Guides
+description: >
+  Establish node to node connections through the Firezone
+  server.A typical use case for this configuration is to enable 
+  an administrator to access a server, container, or machine
+  that is normally behind a NAT or firewall.
+---
+---
+
+This guide will walk through using Firezone as a relay to connect
+two hosts. A typical use case for this configuration is to enable an
```

Comment:
Suggestion: change `hosts` to `peers` to be consistent.

Reply:
updated for consistency/clarity